addons/rook/1.4.9/install.sh

```diff
@@ -211,25 +211,37 @@ function rook_ceph_healthy() {
 
 # rook_version_deployed check that there is only one rook-version reported across the cluster
 function rook_version_deployed() {
+    # wait for our version to start reporting
+    if ! kubectl -n rook-ceph get deployment -l rook_cluster=rook-ceph -o jsonpath='{range .items[*]}{"rook-version="}{.metadata.labels.rook-version}{"\n"}{end}' | grep -q "${ROOK_VERSION}" ; then
```

Comment:
make sure that the version is reporting before checking if multiple versions are reporting to prevent race condition.